pyproject.toml

```diff
@@ -1,2 +1,8 @@
 [build-system]
-requires = ["setuptools", "wheel", "cmake>=3.1.0", "pybind11"]
+requires = [
+    "setuptools>=42",
+    "wheel",
+    "ninja>=1.10",
```

Comment:
Do you really want to require ninja here? I would keep the requirements minimal...

Reply:
Ninja is like CMake, both wheels are packaged by KitWare, so that you'll always have something to build CMake code with. There is no make wheel, and there is no promise that make is always available in Python. 

This was much more elegant when I used Ninja on all platforms, but communicating the correct MSVC was even harder then. I'll retry eventually.